.storybook/test-storybook-canvas.config.js

```diff
@@ -11,6 +11,7 @@ const configuration = {
             'with location, size and segmentBy': DEFAULT_SCENARIO_CONFIG,
             'with location and color': DEFAULT_SCENARIO_CONFIG,
             'with location, size and color': DEFAULT_SCENARIO_CONFIG,
+            'with location, size and color has ignored ratio': DEFAULT_SCENARIO_CONFIG,
```

Comment:
Do we need this storybook test?
IMO, just unit test is enough for this case.
What do you think?

Reply:
@naibinh suggest to me  add more case.

Reply:
It's my bad. I see it's not really necessary here.
Please remove this case from storybook test.

Reply:
removed